index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h3>RACHID</h3>
+
+        <div class="menu"><span></span><span></span><span></span></div>
+      </nav>
+    </header>
+
+    <section class="hero">
+      <h1>I’m Mariane Glad to see you!</h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a href="#">LET’S CONNECT</a>
+      <div class="socials">
+        <img src="./assets/twitter.svg" alt="twitter icon" />
+        <img src="./assets/Linkedin.svg" alt="Linkedin icon" />
+        <img src="./assets/meduim.svg" alt="medium icon" />
+        <img src="./assets/github.svg" alt="github icon" />
+        <img src="./assets/peace.svg" alt="peace icon" />
+      </div>
```

Comment:
- [ ] Good job in getting these social icons and aligning them. Since the parent tag is having a list of children, I think it will be nice if we could use a `ul` tag to group them and a `li` for each icon.

Example:
```
<ul>
  <li></li>
  <li></li>
</ul>
```